client/app/initializers/inject-session-manager.js

```diff
@@ -0,0 +1,8 @@
+export function initialize(application) {
+  application.inject('component', 'session-manager', 'service:session-manager');
```

Comment:
For now I injected the session-manager into ever component, we can expand this as needed.